xml/System/ReadOnlySpan`1.xml

```diff
@@ -422,16 +453,18 @@
         </Attribute>
       </Attributes>
       <ReturnValue>
-        <ReturnType>System.T&amp;</ReturnType>
+        <ReturnType>T@</ReturnType>
       </ReturnValue>
       <Parameters>
         <Parameter Name="index" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <param name="index">The zero-based index of the element to get.</param>
+        <summary>Gets an item from the read-only span at the specified zero-based index.</summary>
+        <value>The element from the read-only span.</value>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.IndexOutOfRangeException">
+          <paramref name="index" /> is less then zero or greater than or equal to <see cref="P:System.SpanReadOnly`1.Length" />.</exception>
```

Comment:
System.SpanReadOnly`1.Length is not formatted correctly in the final document.

Reply:
Probably due to the typo.
SpanReadOnly -> ReadOnlySpan